src/features/burgerMenu/ui/burgerMenu.tsx

```diff
@@ -0,0 +1,16 @@
+import { Menu } from "@headlessui/react";
+
+export function BurgerMenu() {
+  return (
+    <Menu.Items>
+      <Menu.Item>
+        {({ active }) => (
```

Comment:
i`m not sure that active state need here because when navigating through pages, you do not need the active state of links

Reply:
fixed